dessia_common/exports.py

```diff
@@ -41,6 +41,23 @@ def is_builtins_list(list_):
     return True
 
 
+class ExportFormat:
+    def __init__(self, selector: Optional[str], extension: str, method_name: str, text: bool,
+                 export_name: str = "", args: Dict[str, Any] = None):
+        self.selector = selector
+        self.extension = extension
+        self.method_name = method_name
+        self.text = text
```

Comment:
It is not so obvious that text is a boolean. Maybe "with_text" could be better...

Reply:
Yes, I do agree. In my opinion, the best solution would be is_text, is_binary, is_text_like, is_binary_like.

Unfortunately, this has a lot of implications on other repos and would be a breaking change. So we will probably keep it until we have a major version release

Reply:
In fact, I also wanted to change sur "export_name" stuff to "filename" but didn't do it for the same reasons.
The sole purpose of not doing it now is because adding breaking changes by refactoring and adding new feature is a quite dangerous operation. And I already refactored the ExportFormat from a dict to an object while adding selector to it. So I was not willing to take more risks of breaking everything